pypeclub/OpenPype
# Global: fix broken otio review extractor

## Brief description
Fault range creation when source frame 0

## Description
When media source first frame was 0 we didn't need to add 1 frame to duration.

## Testing notes:
1. open your testing project in hiero 
2. add a clip to `main` layer with handles at start 0 end 5 (make sure sequence is having available frames in tail)
3. add video file with no handles and tails under to `review` so it will be reviewable
4. create publishable clip and set workfile start frame to 992 for example and handles 0-5
5. publish with review ON
Frame range should work as expected.
